src/printer.ts

```diff
@@ -264,6 +265,21 @@ export class PugPrinter {
 		return parts;
 	}
 
+	private maybeReplaceTagWithLiteral(search: RegExp, replace: string): void {
```

Comment:
I (personally) don't like functions having `maybe` in it
_Will it do it or will it not?_ :man_shrugging: 
I wan't to be sure when calling a function ^^

Reply:
Well the thing is, it doesn't know if it will or not until it tries… That's how the code so far worked also.

Reply:
I didn't criticize the way it works but the name of the function :slightly_smiling_face:  

Let's go with `replaceTagWithLiteralIfPossible`, IMO that's an okay-ish name

Reply:
Haha no I understand! I wasn't very happy about it either, but couldn't think of a better name at the time. I actually think this whole literal handling could maybe be done in a different way that isn't relying on replacing strings in `this.results` (feels a bit hackish?), and have some thoughts about it already, but first I wanted to fix the actual bugs :)

Reply:
BTW: I'm really enjoying the exchanges here. It's so much more enjoyable than what is usually the case when working on open-source. 👍